src/adapter/tests.rs

```diff
@@ -400,6 +400,116 @@ fn rustdoc_finds_statics() {
     }
 }
 
+#[test]
+fn rustdoc_modules() {
+    let path = "./localdata/test_data/modules/rustdoc.json";
+
+    let content = std::fs::read_to_string(path)
+        .with_context(|| format!("Could not load {path} file, did you forget to run ./scripts/regenerate_test_rustdocs.sh ?"))
+        .expect("failed to load rustdoc");
+
+    let crate_ = serde_json::from_str(&content).expect("failed to parse rustdoc");
+    let indexed_crate = IndexedCrate::new(&crate_);
+    let adapter = Arc::new(RustdocAdapter::new(&indexed_crate, None));
+
+    let mod_query = r#"
+{
+   Crate {
+       item {
+           ... on Module {
+               module: name @output
+               is_crate @output
+               is_stripped @output
+               item {
```

Comment:
Adding `@fold` next to `item` might make this test a bit easier to read — it will aggregate the member item names into a list of strings for each module.